demo/src/style/app.scss

```
@@ -49,7 +49,8 @@ header.navbar {
 .sidebar {
   position: sticky;
   top: 1rem;
-  max-height: 100vh;
+  max-height: CALC(100vh - 4rem);
```

```suggestion
  max-height: calc(100vh - 4rem);
```
I know it works, but UPPERCASE can be seen as an error by reader.

---

Thanks @peterblazejewicz, you're right, I don't know why I used uppercase here :)